rfcs/0145-doc-strings.md

```diff
@@ -0,0 +1,435 @@
+---
+feature: doc-strings
+start-date: 2023-03-27
+author: hsjobeki
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Standard for Doc-strings
+
+Finally distinguish between doc-strings and comments.
+
+# Motivation
+[motivation]: #motivation
+
+This RFC aims at improving consistency of in code documentation. (aka Doc-strings)
+
+The community offers tools and methods (such as nixdoc, nix-doc, etc.) to write and process in-code documentation for functions and other code related atomic expressions.
+This functionality is currently utilized to build a subset of documentation for nix functions. (e.g. nixpkgs.lib documentation via: [nixdoc](https://github.com/nix-community/nixdoc))
+There is also [noogle](https://noogle.dev) that indexes subsets of nixpkgs based on the current comments.
+
+However the format of that __docs-strings__ is itself neither well documented nor standardized.
```

Comment:
mentions of `doc-strings`and  `docs-strings` should be consistent, my recommendation is to use `doc-strings` or `doc-string`s

Reply:
Yep :+1: wasn't on purpose.